bundles/org.openhab.binding.mqtt.generic/src/main/resources/ESH-INF/config/number-channel-config.xml

```diff
@@ -102,5 +102,11 @@
 				converting incoming values (like from '°F' to '°C'). Examples: "°C", "°F"</description>
 			<advanced>true</advanced>
 		</parameter>
+		<parameter name="trigger" type="boolean">
```

Comment:
I'm not sure if it make sense to treat a number channel as a trigger channel, because usually the number represent a state. Can you think about a use case in which a number doesn't reflect a state?

Reply:
You are right.
After thinking about it again, I made the trigger an own channel type.